script.js

```diff
@@ -0,0 +1,55 @@
+class Person {
+  constructor(name, birth, money) {
+    this.name = name;
+    this.birthday = birth;
+    this.money = money;
+    this.info = [`Initial: ${money}`];
+  }
+
+  getInfo() {
+    console.log(`Name: ${this.name}, Age: ${this.__calculateAge(this.birthday)}, Amount: ${this.money}$`)
+  }
+
+  addMoney(amount, info) {
+    this.money += amount;
+    this.info.push(`${info}: ${amount}`);
+  }
+
+  withdrawMoney(amount, info) {
+    this.money -= amount;
+    this.info.push(`${info}: -${amount}`);
+  }
+
+  getAccountHistory() {
+    console.log(this.info);
+  }
+
+  __calculateAge(birthday) {
```

Comment:
new Date(Date.now() - new Date(this.birthday ).getTime()).getFullYear() - new Date(0).getFullYear()